troupon/payment/models.py

```diff
@@ -26,3 +27,18 @@ class TransactionHistory(models.Model):
                                        null=True,
                                        blank=True)
     user = models.ForeignKey(User)
+
+
+class IndividualTransactions(models.Model):
```

Comment:
**Code Quality: Nitpick**

This model should be called `Purchases`. There should be a relationship between the items and the deals model (`deal_id` field perhaps). I should also be able to see an amount for inventory purposes.